engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/ConfluentKafkaAvroSerialization.scala

```diff
@@ -0,0 +1,16 @@
+package pl.touk.nussknacker.engine.avro.schemaregistry.confluent.serialization
+
+import org.apache.avro.Schema
+import org.apache.kafka.common.errors.SerializationException
+import pl.touk.nussknacker.engine.avro.AvroUtils
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client.ConfluentSchemaRegistryClient
+
+private[serialization] trait ConfluentKafkaAvroSerialization {
```

Comment:
Why it is splitted from `ConfluentKafkaAvroDeserializer`? It doesn't help when deserializer extends serialization :)

Reply:
I will use it in future at Serializer, so I've extracted it to `ConfluentKafkaAvroSerialization` :)